mws/mws.go

```diff
@@ -61,17 +61,17 @@ type Network struct {
 
 // AwsKeyInfo has information about the KMS key for BYOK
 type AwsKeyInfo struct {
-	KeyArn    string `json:"key_arn,omitempty"`
-	KeyAlias  string `json:"key_alias,omitempty"`
-	KeyRegion string `json:"key_region,omitempty"`
+	KeyArn    string `json:"key_arn"`
+	KeyAlias  string `json:"key_alias"`
+	KeyRegion string `json:"key_region,omitempty" tf:"computed"`
 }
 
 // CustomerManagedKey contains key information and metadata for BYOK for E2
 type CustomerManagedKey struct {
```

Comment:
I'ld also suggest moving these two to `resource_customer_managed_key.go`

Reply:
resolved